$NetBSD: patch-ad,v 1.4 2013/02/10 08:39:00 ryoon Exp $

regen after patching configure.in:

- use export-dynamic correctly (first hunk)
- check for sched_get_priority_max being in -lrt

--- configure.orig	2012-10-11 19:45:04.000000000 +0000
+++ configure
@@ -18085,7 +18085,7 @@ $as_echo "no" >&6; }
 
 fi
 
-if test "x$PKG_CONFIG" == "x"
+if test "x$PKG_CONFIG" = "x"
 then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: No pkg-config utility found or it's too old, I will have trouble finding installed libraries." >&5
 $as_echo "$as_me: WARNING: No pkg-config utility found or it's too old, I will have trouble finding installed libraries." >&2;}
@@ -24165,6 +24165,72 @@ eval as_val=\$$as_ac_var
 #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
 _ACEOF
 
+else
+  { echo "$as_me:$LINENO: checking for sched_get_priority_max in -lrt" >&5
+echo $ECHO_N "checking for sched_get_priority_max in -lrt... $ECHO_C" >&6; }
+if test "${ac_cv_lib_rt_sched_get_priority_max+set}" = set; then
+  echo $ECHO_N "(cached) $ECHO_C" >&6
+else
+  ac_check_lib_save_LIBS=$LIBS
+LIBS="-lrt  $LIBS"
+cat >conftest.$ac_ext <<_ACEOF
+/* confdefs.h.  */
+_ACEOF
+cat confdefs.h >>conftest.$ac_ext
+cat >>conftest.$ac_ext <<_ACEOF
+/* end confdefs.h.  */
+
+/* Override any GCC internal prototype to avoid an error.
+   Use char because int might match the return type of a GCC
+   builtin and then its argument prototype would still apply.  */
+#ifdef __cplusplus
+extern "C"
+#endif
+char sched_get_priority_max ();
+int
+main ()
+{
+return sched_get_priority_max ();
+  ;
+  return 0;
+}
+_ACEOF
+rm -f conftest.$ac_objext conftest$ac_exeext
+if { (ac_try="$ac_link"
+case "(($ac_try" in
+  *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
+  *) ac_try_echo=$ac_try;;
+esac
+eval "echo \"\$as_me:$LINENO: $ac_try_echo\"") >&5
+  (eval "$ac_link") 2>conftest.er1
+  ac_status=$?
+  grep -v '^ *+' conftest.er1 >conftest.err
+  rm -f conftest.er1
+  cat conftest.err >&5
+  echo "$as_me:$LINENO: \$? = $ac_status" >&5
+  (exit $ac_status); } && {
+	 test -z "$ac_c_werror_flag" ||
+	 test ! -s conftest.err
+       } && test -s conftest$ac_exeext &&
+       $as_test_x conftest$ac_exeext; then
+  ac_cv_lib_rt_sched_get_priority_max=yes
+else
+  echo "$as_me: failed program was:" >&5
+sed 's/^/| /' conftest.$ac_ext >&5
+
+	ac_cv_lib_rt_sched_get_priority_max=no
+fi
+
+rm -f core conftest.err conftest.$ac_objext conftest_ipa8_conftest.oo \
+      conftest$ac_exeext conftest.$ac_ext
+LIBS=$ac_check_lib_save_LIBS
+fi
+{ echo "$as_me:$LINENO: result: $ac_cv_lib_rt_sched_get_priority_max" >&5
+echo "${ECHO_T}$ac_cv_lib_rt_sched_get_priority_max" >&6; }
+if test $ac_cv_lib_rt_sched_get_priority_max = yes; then
+  EXTRA_LIBS="$EXTRA_LIBS -lrt"
+fi
+
 fi
 done
 
@@ -25078,6 +25144,7 @@ if ac_fn_c_try_link "$LINENO"; then :
   mp_cv_ncursesw=yes
 else
   mp_cv_ncursesw=no
+  LIBS="$mp_save_LIBS"
 fi
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
@@ -27422,7 +27489,7 @@ if test "${with_vorbis+set}" = set; then
 fi
 
 
-if test "x$with_vorbis" == "xtremor"
+if test "x$with_vorbis" = "xtremor"
 then
 
 pkg_failed=no
